<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1165</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1141</width>
     <height>751</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>LED Board</string>
    </attribute>
    <widget class="QListWidget" name="lbProblemList">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>411</width>
       <height>391</height>
      </rect>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>440</y>
       <width>411</width>
       <height>191</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pbLogProblem">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Log Problem</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbStars">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>85</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>*</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>**</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>***</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cbGrade">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>85</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>7a</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6c+</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6c</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6b+</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6b</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6a+</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6a</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QPushButton" name="pbMirror">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mirror</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="tbProblemInfo">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>30</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbUsers">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>240</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbViewUsers">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>240</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>View User</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbTestLEDs">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>670</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Test LEDs</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Add Problem</string>
    </attribute>
    <widget class="QPushButton" name="pb1">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>550</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pb2">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>550</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb3">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>550</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb4">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>550</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb5">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>550</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb6">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>550</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb7">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>550</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb8">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>550</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>630</y>
       <width>671</width>
       <height>35</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pbDiscard">
        <property name="text">
         <string>Discard</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leProblemName"/>
      </item>
      <item>
       <widget class="QComboBox" name="cbGrade_2">
        <item>
         <property name="text">
          <string>7a</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6c+</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6c</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6b+</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6b</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6a+</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6a</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbStars_2">
        <item>
         <property name="text">
          <string>*</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>**</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>***</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>pb1</zorder>
    <zorder>pb2</zorder>
    <zorder>pb3</zorder>
    <zorder>pb4</zorder>
    <zorder>pb5</zorder>
    <zorder>pb6</zorder>
    <zorder>pb7</zorder>
    <zorder>pb8</zorder>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Users</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
